docs/src/test-webserver-js.md

```diff
@@ -32,8 +32,11 @@ export default defineConfig({
 | `command`| Shell command to start the local dev server of your app. |
 | `url`| URL of your http server that is expected to return a 2xx, 3xx, 400, 401, 402, or 403 status code when the server is ready to accept connections. |
 | `reuseExistingServer`| If `true`, it will re-use an existing server on the url when available. If no server is running on that url, it will run the command to start a new server. If `false`, it will throw if an existing process is listening on the url. To see the stdout, you can set the `DEBUG=pw:webserver` environment variable. |
+| `ignoreHTTPSErrors` | Whether to ignore HTTPS errors when fetching the `url`. Defaults to `false`. |
```

Comment:
Why do we have these options documented in 2 places, can we delete this one and refer to the API docs to avoid such bugs in the future?

Reply:
+ that its only in a central place
- that its hard to link to the api docs, since they would have to scroll down.

I'd keep it for now as it is, less clicks required are for the users.